src/providers/irc/AbstractIrcServer.cpp

```diff
@@ -267,14 +267,14 @@ ChannelPtr AbstractIrcServer::getChannelOrEmpty(const QString &dirtyChannelName)
 std::vector<std::weak_ptr<Channel>> AbstractIrcServer::getChannels()
 {
     std::lock_guard lock(this->channelMutex);
-    std::vector<std::weak_ptr<Channel>> channels;
+    std::vector<std::weak_ptr<Channel>> rchannels;
```

Comment:
Rename to something that doesn't have a pointless prefix like `result`

Reply:
FWIW, the warning was:

`warning C4458: declaration of 'channels' hides class member`

Reply:
Do you have a preference on name here?